### ISSUE TITLE
Music2 bug in "music2_prop_t_statistics" function

### ISSUE THREAD

**Issue by ste-depo** [xuranw/MuSiC#121]
Error:
```
Error in (function (classes, fdef, mtable)  : 
  unable to find an inherited method for function ‘exprs’ for signature ‘"matrix"’
```
Raised from:
https://github.com/xuranw/MuSiC/blame/0a3e3af45d4bd018939013660a3e83e580fa3bac/R/music2.R#L340

The actual code tries to extract expression from "bulk.case" pretending it is an object of ExpressionSet class, while it's  already a matrix. 

Removing exprs() fixes the bug

**Comment by marounbs** [comment]
Yes. Reached the same conclusion, and lost a couple of hours. Too bad I didn't see your post first.

**Comment by Lindsey8383** [comment]
This was useful for me to address the issue in R.

`
install.packages("git2r")
library(git2r)
repo_url <- "https://github.com/xuranw/MuSiC.git"
local_dir <- file.path(.libPaths()[1], "MuSiC_download")
clone(repo_url, local_dir)

# make a local change to address a bug in the code
file2change <- file.path(local_dir, "R", "music2.R")
# read the contents of the file
file_lines <- readLines(file2change)
# define the new line content
new_line <- "case_s = bulk.case[exp_genel, colnames(bulk.case) %in% id_d]"
# modify the specific line (line number 340 in this case)
line_number_to_change <- 340
file_lines[line_number_to_change] <- new_line
# write the modified content back to the file (overwrite)
writeLines(file_lines, file2change)

# install modified package
devtools::install_local(local_dir, force = TRUE)`

**Comment by alexandriatran** [comment]
Thanks for documenting. I just ran into this issue using MuSiC installed from tutorial instructions, so it has not yet been resolved on that branch.

**Comment by ahgillmo** [comment]
Has anyone found a solution for this issue yet?

**Comment by DavideBrex** [comment]
Found same problem, still not resolved

**Comment by nroak** [comment]
Here to report the same issue.
